contracts/EverlongStrategy.sol

```diff
@@ -793,6 +783,35 @@ contract EverlongStrategy is BaseStrategy {
             IHyperdrive(hyperdrive).isMature(_portfolio.head());
     }
 
+    /// @notice Gets the minimum amount of strategy assets needed to open a long
+    ///         with hyperdrive.
+    /// @return amount Minimum amount of strategy assets needed to open a long
+    ///                with hyperdrive.
+    function minimumTransactionAmount() public view returns (uint256 amount) {
+        // Retrieve the minimum transaction amount from the poolConfig.
+        // This value is already converted to shares if `asBase==false`.
+        amount = IHyperdrive(hyperdrive)._minimumTransactionAmount(
+            _poolConfig,
+            asBase
+        );
+
+        // If we're using a wrapped token, make sure to convert to the unwrapped
+        // value.
+        if (isWrapped) {
+            amount = convertToUnwrapped(
+                IHyperdrive(hyperdrive)._minimumTransactionAmount(
+                    _poolConfig,
+                    asBase
+                )
```

Comment:
this does a `_convertToBase` again.
In the example, don't we have `strategy.asset() == wstEth` and `executionToken == stEth` but `asBase = false`?
if we want the result in strategy assets (wstEth), then the `amount` with `asBase = false` in 739 is already in correct units? I don't think we need this wrapping here.

Reply:
Yup! Correct, great catch